runtime/compiler/runtime/GPUHelpers.cpp

```diff
@@ -842,8 +842,8 @@ cudaInfoHashPut(CudaInfo *cudaInfo, void **hostref, int32_t elementSize, CUdevic
 
       cudaInfo->hashSize = hashSize;
       cudaInfo->hashTable = hashTable;
-
-      memcpy(cudaInfo->hashTable, oldHashTable, oldHashSize * sizeof(HashEntry));
+      // Makes no sense hashTable to oldHashTable in 2 lines???
+      cudaInfo->hashTable = oldHashTable
```

Comment:
This was correct before (`oldHashTable` is not large enough, and it's freed on line 848).